### DISCUSSION TITLE
update package version on pypi.org

### DISCUSSION THREAD

**Discussion by mvzakharov** [vertica/dbt-vertica#142] [Q&A] [answered]
Hi guys. We are using your code, thank you very much for your work!
Recently we encountered a problem with updating the version of the package, because you have a #140  for version 1.8.5, but the site pypi.org indicates the old version 1.7.3.
https://pypi.org/project/dbt-vertica/1.7.3/

Do you plan to update the version on the site pypi.org?

**Comment by ajay-abrol2** [comment, marked as answer]
Hi,
Working on to publish latest version 1.8.5 to pypi website and will update you once complete.

Thanks & Regards,
Ajay Abrol
